app/locations/middleware.test.js

```diff
@@ -266,26 +266,57 @@ describe('Locations middleware', function () {
         session: {},
       }
       nextSpy = sinon.spy()
-      mockReferenceData = {}
     })
 
     context('when the API is available', function () {
-      it('should retrieve all regions', function () {
-        mockReferenceData.getRegions = sinon.stub().returns(Promise.resolve([]))
-        proxiedMiddleware.setRegions(req, {}, nextSpy)
-        expect(req.session).to.have.property('regions')
-        expect(req.session.regions).to.deep([])
+      it('should retrieve all regions', async function () {
+        mockReferenceData.getRegions = sinon.fake.returns(Promise.resolve([]))
```

Comment:
`sinon.fake.resolves([])` ?

Reply:
`getRegions` is an `async` function, I thought I could keep the same return type.
Happy to change it.

Reply:
Fair enough.